packages/stark-ui/src/modules/dropdown/components/dropdown.component.spec.ts

```diff
@@ -561,6 +561,47 @@ describe("DropdownComponent", () => {
 				});
 			});
 
+			describe("DropDown properties", () => {
+				let mockObserver: SpyObj<Observer<any>>;
+
+				beforeEach(() => {
+					mockObserver = createSpyObj<Observer<any>>("observerSpy", ["next", "error", "complete"]);
+					component = hostComponent.dropdownComponent;
+				});
+
+				it("the MatOption should be disabled when the form control is disabled AND it should trigger a 'valueChange' event ONLY IF the 'emitEvent' option is enabled", () => {
```

Comment:
Could you please split this test in 2 parts (one with `emitEvent: false`, the other with `emitEvent: true` ?
Also, could you rephrase the `it` ?  
What you want to test here is that when you disable the `formControl` with(out) `emitEvent`, it reacts as expected. 
You don't care at this moment if the MatOption is disabled or not. Actually this is checked already at line 552 😊